src/main.js

```diff
@@ -61,10 +65,22 @@ window.addEventListener("DOMContentLoaded", () => {
     // Navbar
     initializeNavbar();
 
+    // Footer FormIO Action
+    // Note: This is added here, as there is an issue with breadcrumbShorten() function.
+    //       Will move this once that issue is fixed.
+    const footerFormio = document.getElementById("qg-feedback-toggle");
+    if (footerFormio) {
+      document.getElementById("btn-footer-feedback").addEventListener("click", displayFeedbackForm);
+    }
+
     // Breadcrumb
     initBreadcrumb();
 
     // Quick exit
+    window.addEventListener("scroll", positionQuickExit, true);
```

Comment:
could these functions be moved into the initQuickexit() or positionQuickExit() setup functions?